xml/alerts-2.5.rng

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<grammar xmlns="http://relaxng.org/ns/structure/1.0" 
+         datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
+  <start>
+      <ref name="element-alerts"/>
+  </start>
+
+  <define name="element-alerts">
+    <optional>
+    <element name="alerts">
+      <zeroOrMore>
+        <element name="alert">
+          <attribute name="id"><text/></attribute>
+          <!-- path to the script called for alert -->
+          <attribute name="path"><text/></attribute>
+          <optional>
+            <attribute name="timeout"><text/></attribute>
+          </optional>
+          <optional>
+            <!-- format string fed into strftime defaults to %s -->
+            <!-- use format as with strftime + %N as with date -->
+            <attribute name="tstamp_format"><text/></attribute>
+          </optional>
+          <zeroOrMore>
+            <element name="environment_variables">
```

Comment:
Well I was searching for something more catchy than "environment_variables" but lacking a 
good idea I tended to calling it what it is.
I was a little afraid of reusing instance_attributes & meta_attributes for something similar but
not quite the same. But you are probably right that the use here is close enough.
But then I would allow anything to be overruled by instance_attributes & meta_attributes in
the recipient sections. Something like a different timeout for a far-away-recipient (in case it
is not just an email-address but a network address that is being contacted) might be
useful as well and if we have the same mimic for all attributes source of confusion is mitigated.